middle_end/flambda2.0/to_cmm/un_cps.ml

```diff
@@ -1193,23 +1197,30 @@ let unit (unit : Flambda_unit.t) =
             Flambda_kind.value;
         ]
       in
-      let _return_cont, env =
+      let return_cont, env =
         Env.add_jump_cont env (List.map snd return_cont_params)
           (Flambda_unit.return_continuation unit)
       in
-      (* let functions = program_functions offsets used_closure_vars unit in *)
-      let _body = expr env (Flambda_unit.body unit) in
-      Misc.fatal_error "To be continued"
-(*
+      let body = expr env (Flambda_unit.body unit) in
       let body =
         let unit_value = C.targetint Targetint.zero in
```

Comment:
This should be the targetint `1` (i.e. `0` tagged), rather than `0` in order to be `unit` no ?

Reply:
Fixed